spec/controllers/cloud_network_controller_spec.rb

```diff
@@ -249,36 +237,37 @@
       controller.instance_variable_set(:@_params, params)
     end
 
-    context 'tagging instances from a list of instances, accessed from the details page of a network' do
+    context 'when tagging instances from a list of instances, accessed from the details page of a network' do
       let(:params) { {:pressed => "instance_tag"} }
 
       it 'calls tag method for tagging instances' do
         expect(controller).to receive(:tag).with("VmOrTemplate")
+
         controller.send(:button)
       end
     end
 
-    context 'tagging network routers from a list of routers, accessed from the details page of a network' do
+    context 'when tagging network routers from a list of routers, accessed from the details page of a network' do
       let(:params) { {:pressed => "network_router_tag"} }
 
       it 'calls tag method for tagging network routers' do
         expect(controller).to receive(:tag).with("NetworkRouter")
+
         controller.send(:button)
       end
     end
   end
 
   describe "#delete_networks" do
     let(:admin_user) { FactoryGirl.create(:user, :role => "super_administrator") }
-    let(:network) { FactoryGirl.create(:cloud_network) }
+
     before do
-      EvmSpecHelper.create_guid_miq_server_zone
       login_as admin_user
       controller.instance_variable_set(:@_params, :id => network.id, :pressed => 'cloud_network_delete')
+      allow(controller).to receive(:process_cloud_networks).with([network], "destroy")
     end
 
-    it "testing " do
-      allow(controller).to receive(:process_cloud_networks).with([network], "destroy")
+    it do
```

Comment:
Never have an empty `it`, no one will understand what it does. Also I'm missing the `expect` calls after the method call...